tools/building.py

```diff
@@ -1357,20 +1356,23 @@ def map_to_js_libs(library_name):
     'pthread': [],
     'X11': ['library_xlib.js'],
     'SDL': ['library_sdl.js'],
-    'stdc++': [],
     'uuid': ['library_uuid.js'],
     'websocket': ['library_websocket.js']
   }
+  # And some are hybrid and require JS and native libraries to be included
+  native_library_map = {
+    'GL': 'libgl',
```

Comment:
will this conflict with the other PR that renames it to libGL?

Reply:
Yes, I plan to land this one first.